config/locales/de.yml

```diff
@@ -1,61 +1,147 @@
 ---
 de:
+  aaa_injectable: &id-injectables
```

Comment:
Hmm why the `aaa_injectable`? It looks like you're only including it once below `de`.

Reply:
The reason for this is that keys to be injected have always to be defined higher in the yaml file than when there injected with `*id-my-injected-key`
With this I intend to make sure that even sorting the yaml file will ensure those keys to be at the very beginning.

Reply:
Ah ok, makes sense.